src/lib.rs

```diff
@@ -544,6 +544,11 @@ pub struct BindgenOptions {
     pub whitelist_recursively: bool,
 }
 
+/// TODO(emilio): This is sort of a lie (see the error message that results from
+/// removing this), but since we don't share references across panic boundaries
+/// it's ok.
+impl ::std::panic::UnwindSafe for BindgenOptions {}
```

Comment:
I believe this and the change in `TypeChooser` is not needed if we do the following.